workflow.txt

```diff
@@ -1,10 +1,9 @@
+fork on github
 git clone
 git checkout -b my-solution
-git commit -m ""
-git fetch --all
 git add .
-git pull upstream master
-git merge my-solution
-fork on github
+git commit -m ""
 git checkout master
+git merge my-solution
 git push origin my-solution
+git pull upstream master
```

Comment:
👍 if working with others who may have changed master, we'll want to pull master and merge my solution first (to make sure the are no merge conflicts). Will discuss team workflows in much more detail before project 3